_includes/assets/sass/components/_home-page.scss

```diff
@@ -1,12 +1,14 @@
 #home-page {
     #design-sprint {
-        @media (min-width: $tbds-breakpoint-medium) {
-          max-width: 60ch;
-        }
+        // REMOVE - was causing issues on design sprint and re-introduced as a utility class.
+
+        // @media (min-width: $tbds-breakpoint-medium) {
+        //   max-width: 60ch;
+        // }
```

Comment:
Should we remove this commented out code?

Reply:
Yup! Went ahead and removed it.